src/main/resources/templates/user/settings.html

```diff
@@ -17,6 +17,12 @@ <h3>Instructions for getting API Key</h3>
                     Click Here to get API Key
                   </a>
                 </div> 
+                
+                 <div class="container" th:if="${isLoggedIn} and ${isAdmin}">
+                  <h3>Oauth.toString()</h3>
```

Comment:
Is this for debug purposes? This will literally display a header with the text "Oauth.toString()"